static/app/views/performance/utils/useInsightsURL.tsx

```diff
@@ -0,0 +1,30 @@
+import useOrganization from 'sentry/utils/useOrganization';
+import {ModuleName} from 'sentry/views/starfish/types';
+
+type ModuleNameStrings = `${ModuleName}`;
+type RoutableModuleNames = Exclude<ModuleNameStrings, '' | 'other'>;
+
+export function useInsightsURL(moduleName: RoutableModuleNames) {
+  const builder = useInsightsURLBuilder();
+  return builder(moduleName);
+}
+
+type URLBuilder = (moduleName: RoutableModuleNames) => string;
+
+export function useInsightsURLBuilder(): URLBuilder {
+  const organization = useOrganization({allowNull: true}); // Some parts of the app, like the main sidebar, render even if the organization isn't available (during loading, or at all).
+
+  if (!organization) {
+    // If there isn't an organization, items that link to modules won't be visible, so this is a fallback just-in-case, and isn't trying too hard to be useful
+    return () => '';
+  }
```

Comment:
Do we need this conditional? Isn't the fallback behaviour on lines 26-28 correct even if organization is not defined?

Reply:
True! I copy-pasted this from `useModuleURL`! Removed